lib/api.js

```diff
@@ -1614,9 +1616,6 @@ function wrapLambdaHandler (userHandler) {
       finalize = (span) => {
         span.defaultTxName = process.env.APPOPTICS_TRANSACTION_NAME || `${method.toUpperCase()}.${userHandlerName}`;
```

Comment:
Would we end up with a transaction name like `.handlerName` if there is no method?

Reply:
it will end up as `undefined.handlerName`. thoughts?

Reply:
fixed this and now uses lambda-function name, not handler function name.